src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/services/email/EmailSenderTest.java

```diff
@@ -60,22 +87,43 @@ void should_send_email_to_all_recepients() throws Exception {
             Map.of(FILE_NAME_1, file1, FILE_NAME_2, file2)
         );
 
-        MimeMessageParser msg = new MimeMessageParser(greenMail.getReceivedMessages()[0]).parse();
-
-        assertThat(msg.getFrom()).isEqualTo(FROM_ADDRESS);
-        assertThat(msg.getTo())
-            .extracting(Address::toString)
-            .hasSize(2)
-            .containsExactly(
-                RECIPIENT_1,
-                RECIPIENT_2
+        greenMail.waitForIncomingEmail(RECIPIENTS.length);
+
+        assertThat(greenMail.getReceivedMessages()).hasSize(RECIPIENTS.length);
+        MimeMessage msg = greenMail.getReceivedMessages()[0];
+        String headers = GreenMailUtil.getHeaders(msg);
+        assertThat(headers.contains("From: " + FROM_ADDRESS)).isTrue();
+        assertThat(headers.contains("To: " + RECIPIENT_1 + ", " + RECIPIENT_2)).isTrue();
+        assertThat(headers.contains("Subject: " + SUBJECT)).isTrue();
+        assertThat(msg.getContentType().startsWith(MULTIPART_MIXED_VALUE)).isTrue();
```

Comment:
you can use `assertThat(msg.getContentType()).startsWith(MULTIPART_MIXED_VALUE)`
with will result in a much better test error message in case of a failure